### PR TITLE
chore: switched to Node@18 (NPM 9)

### PR DESCRIPTION
**Pull request by kirrg001**
- nvm install 18.14.0
- nvm alias default 18.14.0
- nvm use default

From https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion
> 3: The lockfile version used by npm v7, without backwards compatibility affordances. This is used for the hidden lockfile at node_modules/.package-lock.json, and will likely be used in a future version of npm, once support for npm v6 is no longer relevant.

- updated some lock files to use version 3 already (apps)
- removed lockFileVersion: 1 in separate commit

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,6 @@
     "test:debug": "lerna run test:debug --stream",
     "typecheck": "tsc",
     "update-deps": "lerna exec '../../node_modules/.bin/ncu --upgrade' && lerna exec 'npm i --package-lock-only' && npm run reinstall-deps",
-    "verify": "lerna run verify --stream",
```
Comment (basti1302) [inline package.json:53]: _question:_ What's up with this deletion?

**Review by basti1302 — APPROVED** [review]
Good stuff!

File: package.json (hunk 1 above)
Comment (kirrg001) [reply package.json:53]: This was a mistake.

**Comment by kirrg001** [comment]
FYI Node v10 build for msgpackr-extract was already failing on main
See e.g. https://app.circleci.com/pipelines/github/instana/nodejs/5003/workflows/2cd78bdf-8473-41a4-b53e-9f1a686dd479/jobs/18550

As this is an optional dependency, **it is fine that it fails** for Node v10. We 4.10.2 has dropped support for Node v10.

So why is the build now failing with Node v10?
Exit code 137 is memory issue again.

**Pull request merged**